velox/common/base/BitUtil.h

```diff
@@ -118,6 +118,18 @@ inline int32_t getAndClearLastSetBit(uint16_t& bits) {
   return trailingZeros;
 }
 
+#if __ARM_ARCH
+inline int32_t arm64_getAndClearLastSetBit(uint64_t& bits) {
+  int32_t trailingZeros = __builtin_ctzll(bits) >> 2;
```

Comment:
It is cleaner to just return the number of trailing zeros as it is and divided by 4 outside the call